csharp/ql/src/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+## 0.0.4
```

Comment:
Same suggestion as in the other PR. Add a line entry even though there were no changes.